var/spack/repos/builtin/packages/lm-sensors/package.py

```diff
@@ -31,6 +31,14 @@ class LmSensors(MakefilePackage):
     depends_on("flex", type="build")
     depends_on("perl", type="run")
 
+    @property
+    def build_targets(self):
+        targets = []
+
+        targets.append("CC={0}".format(self.compiler.cc))
```

Comment:
```suggestion
        targets.append("CC={0}".format(spack_cc))
```

Sorry for the late review, but you wanna use the (awkward) spack_cc global, which points to the compiler wrapper, whereas `self.compiler.cc` points to the underlying compiler.